src/pe/export.rs

```diff
@@ -195,6 +189,7 @@ impl<'a> ExportData<'a> {
             file_alignment,
             opts,
         )
+        .ok()
```

Comment:
The call to `Ok()` drops the error. Returning the error is a better idea. You can use `Result::and_then()` instead of `Option::and_then()`, or instead use the `?` operator.